media.go

```diff
@@ -145,6 +145,14 @@ var mediaTypeMap = map[MediaType]string{
 	MediaAudio:    "/mms/audio",
 }
 
+// Upload only when media is nill
+func (wac *Conn) uploadMedia(m *Media, reader io.Reader, appInfo MediaType) (downloadURL string, mediaKey []byte, fileEncSha256 []byte, fileSha256 []byte, fileLength uint64, err error) {
+	if m == nil {
```

Comment:
It sounds weird when we have to pass a nil Media expecting that will be a new upload.

Reply:
This is an internal function a wrapper over the original function, for new uploads the original method can be used anyways